index.html

```diff
@@ -102,25 +102,25 @@ <h3><del>Что посмотреть на выходных: &#128293; </del></h3
   <fieldset>
     <legend>My first form</legend>
     <label for="name">Name</label>
-    <div><input id="name" type="text" placeholder="Name" required></div>
+    <div><input name="name" id="name" type="text" placeholder="Name" required></div>
     <label for="email">Email</label>
-    <div><input id="email" type="email" required></div>
+    <div><input name="email" id="email" type="email" required></div>
     <label for="password">Password</label>
-    <div><input id="password" type="password" required></div>
+    <div><input name="password" id="password" type="password" required></div>
     <label for="range">Your programming level</label>
-    <div><input id="range" min="1" max="10" type="range"></div>
+    <div><input name="range" id="range" min="1" max="10" type="range"></div>
     <p><label>What does HTML stand for?</label></p>
-    <div><input id="box1" type="checkbox" value="Hyperlinks and text Markup language"><label for="box1">Hyperlinks and text Markup language</label></div>
-    <div><input id="box2" type="checkbox" value="Home Tool Markup Language"><label for="box2">Home Tool Markup Language</label></div>
-    <div><input id="box3" type="checkbox" value="Hyper Text Markup Language"><label for="box3">Hyper Text Markup Language</label></div>
+    <div><input name="box1" id="box1" type="checkbox" value="Hyperlinks and text Markup language"><label for="box1">Hyperlinks and text Markup language</label></div>
+    <div><input name="box2" id="box2" type="checkbox" value="Home Tool Markup Language"><label for="box2">Home Tool Markup Language</label></div>
+    <div><input name="box3" id="box3" type="checkbox" value="Hyper Text Markup Language"><label for="box3">Hyper Text Markup Language</label></div>
     <p><label>What does HTTP stand for?</label></p>
-    <div><input id="rad1" type="radio"><label for="rad1">Hyper Text Translate Protocol</label></div>
-    <div><input id="rad2" type="radio"><label for="rad2">Hyper Text Transmission Protocol</label></div>
-    <div><input id="rad3" type="radio"><label for="rad3">Hyper Text Transfer Protocol</label></div>
+    <div><input name="rad1" id="rad1" type="radio"><label for="rad1">Hyper Text Translate Protocol</label></div>
+    <div><input name="rad2" id="rad2" type="radio"><label for="rad2">Hyper Text Transmission Protocol</label></div>
+    <div><input name="rad3" id="rad3" type="radio"><label for="rad3">Hyper Text Transfer Protocol</label></div>
```

Comment:
you set different names here - so, try to check all radio btns and you will see a bug =)